### PR TITLE
Add a Modern PCL project that also support CoreClr

### PR DESCRIPTION
**Pull request by jeremymeng**
For #185 
This is mostly a copy of the old portable csproj, with different targeted profile.

### CODE REVIEW

File: Portable.Ninject.nuspec [hunk 1]
```diff
@@ -19,6 +19,22 @@
     <summary>IoC container for .NET</summary>
     <releaseNotes>Xamarin support for iOS and Android</releaseNotes>
     <tags>Ninject ioc di portable pcl xamarin ios android monotouch monoandroid</tags>
+    <dependencies>
+      <group targetFramework="dotnet">
```
Comment (onovotny) [inline Portable.Ninject.nuspec]: I would recommend using `NuSpec.ReferenceGenerator` to add/update the dependencies as they can change over time and the tool will ensure they're up-to-date.

**Comment by onovotny** [comment]
Can you please add a unit test project for DNX as well?

File: Portable.Ninject.nuspec (hunk 1 above)
Comment (jeremymeng) [inline Portable.Ninject.nuspec]: I actually tried. But for some reason it didn't work. `NuSpec.ReferenceGenerator` wasn't getting added to the `project.json`.  I didn't have time to look into it yet.

**Comment by jeremymeng** [comment]
Thanks for asking to add the test project.  It revealed that I cannot just simply copy the PCL project.

**Comment by jeremymeng** [comment]
Looks like the build machine doesn't support .Net 4.6.

File: Ninject.sln [hunk 2]
```diff
@@ -424,6 +431,54 @@ Global
 		{420ACA2A-7853-4C5D-A3B1-E7AE938A801F}.Release|Mixed Platforms.ActiveCfg = Release|Any CPU
 		{420ACA2A-7853-4C5D-A3B1-E7AE938A801F}.Release|Mixed Platforms.Build.0 = Release|Any CPU
 		{420ACA2A-7853-4C5D-A3B1-E7AE938A801F}.Release|x86.ActiveCfg = Release|Any CPU
+		{00A3D52F-BA8F-49A3-BDF8-FA457E866F3A}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```
Comment (jeremymeng) [inline Ninject.sln]: I don't understand why adding a Modern PCL project creates so many configurations.

**Comment by onovotny** [comment]
Working on the build machine. AppVeyor is setup but the current build server is interfering
https://ci.appveyor.com/project/onovotny/ninject

**Comment by jeremymeng** [comment]
@onovotny is there anything I can do to move this PR forward?

**Comment by ah-** [comment]
What's the status of this? I think the new infrastructure has now sufficiently stabilized to start building netstandard compatible packages.

A starting point might be https://github.com/ah-/Ninject/commit/b6820cb2f13f98fd9795a56b525a1a340d263652, would you accept an updated PR?

**Comment by jeremymeng** [comment]
Closing as the master branch already has netstandard support.

**Pull request closed without merging**